inference-engine/src/mkldnn_plugin/nodes/mkldnn_one_hot_node.cpp

```diff
@@ -76,6 +76,8 @@ MKLDNNOneHotNode::MKLDNNOneHotNode(const std::shared_ptr<ngraph::Node>& op, cons
     if (!( ((1 + src_dims.size()) == dst_dims.size()) ||
            (src_dims.size() == 1 && dst_dims.size() == 1 && dst_dims[0] == depth && src_dims[0] == 1)))
         IE_THROW() << errorPrefix << " has incorrect number of input/output dimensions!";
+
+    output_precision = getOriginalOutputPrecisionAtPort(0);
```

Comment:
That's not valid change. We have to specify precision inside `initSupportedPrimitiveDescriptors` since it can be change after the ctor during BF16 converter

Reply:
Fixed